src/System.Memory/tests/Span/Clear.cs

```diff
@@ -192,19 +192,30 @@ public static void ClearValueTypeWithReferences()
             Assert.Equal<TestValueTypeWithReference>(expected, actual);
         }
 
-        [ActiveIssue(16492)]
-        [OuterLoop]
         [Fact]
-        public unsafe static void ClearLongerThanUintMaxValueBytes()
+        [OuterLoop]
+        public unsafe static void ClearTests()
+        {
+            // These tests need to be run serially with no risk of them running in parallel
+            // or the sum of the allocations will run the test machine out of memory and result
+            // in failures.
+            // Also, the native version should come before the managed version since the memory
+            // allocation and free are explicit instead of relying on the GC, so we should be
+            // certain memory was freed from the first before the second is called.
+            ClearNativeLongerThanUintMaxValueBytes();
```

Comment:
These tests should have been running serially even before because of XUnit serializes the tests per class by default: https://github.com/xunit/xunit/blob/master/src/xunit.core/CollectionBehavior.cs#L16. This change should be unnecessary.

However, there can still have increased risk of failing if the tests from this class run in parallel with tests in Overflow.cs.

I think we should have a singleton class from which all different tests should checkout the large memory block. It will ensure that we do not have multiple of these large blocks checked out at the same time; and it will also let us tweak the implementation of it as necessary.